src/Resources/config/resources/credit_memo.yaml

```diff
@@ -0,0 +1,6 @@
+sylius_resource:
+  resources:
+    sylius_refund.credit_memo:
+      classes:
+        model: Sylius\RefundPlugin\Entity\CreditMemo
+        repository: BitBag\SyliusMolliePlugin\Repository\CreditMemoRepository
```

Comment:
overwriting a plugin in plugin is not good practice. it is better to extract it and describe it in readme. You can add your repository that you inherit from the original (parent: sylius_refund.repository.credit_memo) without having to overwrite it.